client/my-sites/plans/main.jsx

```diff
@@ -187,6 +188,7 @@ class Plans extends React.Component {
 									site={ selectedSite }
 									plansWithScroll={ false }
 								/>
+								<PerformanceTrackerStop />
```

Comment:
Using the component because `Plans` renders other states we are not interested in measuring